### DISCUSSION TITLE
Air Quality and Temp Sensor Exposed to Home Assistant?

### DISCUSSION THREAD

**Discussion by mikewren** [JeffSteinbok/hass-dreo#148] [Q&A] [answered]
Considering the purchase of a DR-HTF010S - this unit includes an air quality and temperature sensor. 

ondering if these sensors are exposed to Home Assistant - would like to track both over time in Home Assistant.  Thanks!

**Comment by JeffSteinbok** [comment, marked as answer]
Please file an issue to track. List it under new device support.
